CONTRIBUTING.md

```diff
@@ -16,4 +19,19 @@ description: The .NET platform for making web based applications
 
 Wiki content goes below this, and is optional for initial commits.
 
-There is a simple utility for generating tag data in node. It is in the generate.js file. Replace the file name with a CSV file of your choice, and run node generate.
\ No newline at end of file
+There is a simple utility for generating tag data in node. It is in the generate.js file. Replace the file name with a CSV file of your choice, and run node generate.
+
+
+### Legal
+
+You will need to complete a Contributor License Agreement (CLA) before your pull
+request can be accepted. This agreement testifies that you are granting us
+permission to use the source code you are submitting, and that this work is
+being submitted under appropriate license that we can use it.
+
+You can complete the CLA by going through the steps at
+https://cla.microsoft.com. Alternatively, download the agreement
+([Microsoft Contribution License Agreement.pdf](https://opensource.microsoft.com/pdf/microsoft-contribution-license-agreement.pdf)),
+sign, scan, and email it back to <cla@microsoft.com>. Be sure to include your
+GitHub username along with the agreement. Once we have received the signed CLA,
+ we'll review the request. You will only need to do this once.
```

Comment:
@Linkid thanks for adding this

Where did you source the text from?

Reply:
It looks different to me. I'm not sure about the implications of changing this wording. I think it would be best to use it exactly as is without modifying

Reply:
Ok.

Reply:
I didn't get give anyone permission to use the my phone number or email address and I will send you these reports to the correct person to showcase Shanon Glenn Williams is not lying about people being involved in his phone.